Classes/IndexQueue/PageIndexerRequest.php

```diff
@@ -141,7 +141,7 @@ public function send($url) {
 			)
 		));
 
-		$rawResponse = file_get_contents($url, FALSE, $context);
+		$rawResponse = @file_get_contents($url, FALSE, $context);
```

Comment:
But in this case you will get warnings in the backend and this is also not so nice ... you will see the error later in the backend module or devlog.